components/rowListItem.brs

```diff
@@ -26,6 +27,15 @@ end sub
 sub handleFocus()
     scale = 1 + (m.top.focusPercent * 0.15)
     m.itemPoster.scale = [scale, scale]
+    if m.top.focusPercent = 1
+        m.global.mainScreen.backgroundURI = m.itemPoster.uri
+         m.global.mainScreen.HeroDetails = {
+            MainBoldLabel:m.itemTitle.text
+            DateAndDurationLabel:"21 Sep 21 | 3m"
+            DescriptionLabel:m.itemDescription.text
+            CategoryLabel:"World"
+         }
```

Comment:
I feel this area is not formatted correctly. Did you try `Ctrl-Shift-I`? Please make the IDE format on save.